### ISSUE TITLE
Refactor config

### ISSUE THREAD

**Issue by LysianeBouchard** [Ferlab-Ste-Justine/Post-processing-Pipeline#16]
### Description of feature

We would like to align with nf-core architecture for configuration. To do this, we should move some configuration that is currently in nextflow.config in other locations:
- Processes resources such as cpu, memory and disk should go in base.config
- Other module configs such as publish dir, container, extra args, etc. should go in module.config 

Also, the nf-core profiles definitions in nextflow.config are taking a lot of place (ex: docker, conda, charliecloud, apptainer, etc.) . To improve readability, we suggest to move them in a separated file (nf-core-profiles.config).

**Comment by FelixAntoineLeSieur** [comment]
yes, me and Damien also discussed this today. It could probably go in #11 or another PR
